flink-connector-influxdb2/pom.xml

```diff
@@ -39,7 +39,7 @@ under the License.
     <influxdbClient.version>2.0.0</influxdbClient.version>
     <druid.version>0.13.0-incubating</druid.version>
     <!--  Test Properties  -->
-    <testcontainers.version>1.15.2</testcontainers.version>
+    <testcontainers.version>1.16.2</testcontainers.version>
```

Comment:
is this mandatory for flink update?
if not will be better in other commit, only with this update (and related)